src/app/clusters/account-login-server/account-login-server.cpp

```diff
@@ -38,16 +38,17 @@
  *******************************************************************************
  ******************************************************************************/
 
+#include <app/Command.h>
 #include <app/util/af.h>
 
-bool emberAfAccountLoginClusterGetSetupPINCallback(unsigned char *)
+bool emberAfAccountLoginClusterGetSetupPINCallback(chip::app::Command * apCommandObj, unsigned char *)
```

Comment:
`ap` prefix is not the style that is used in those files. Please call it `commandObj` or something like that to stay consistent with the style used in those files. For this occurrence and for the others too.